onnx_tf/handlers/backend/conv_integer.py

```diff
@@ -0,0 +1,72 @@
+import tensorflow as tf
+
+from onnx_tf.handlers.backend_handler import BackendHandler
+from onnx_tf.handlers.handler import onnx_op
+from .conv_mixin import ConvMixin
+
+
+@onnx_op("ConvInteger")
+class ConvInteger(ConvMixin, BackendHandler):
+
+  @classmethod
+  def _apply_zero_point(cls, base, zero_point):
+    base = tf.cast(base, tf.float32)
+    zero_point = tf.cast(zero_point, tf.float32)
+    return base - zero_point
+
+  @classmethod
+  def version_10(cls, node, **kwargs):
+    tensor_dict = kwargs["tensor_dict"]
+    x = tensor_dict[node.inputs[0]]
+    w = tensor_dict[node.inputs[1]]
+
+    def process_conv(new_x, new_w):
+      # Remove zero-points from inputs
+      if "x_zero_point" in node.inputs:
+        node.inputs.remove("x_zero_point")
+      if "w_zero_point" in node.inputs:
+        node.inputs.remove("w_zero_point")
+
+      tensor_dict[node.inputs[0]] = new_x
+      tensor_dict[node.inputs[1]] = new_w
+
+      # Use common conv handling
+      conv_node = cls.conv(node, kwargs["tensor_dict"])
+      return conv_node
+
+    # Apply x_zero_point first
+    x = cls._apply_zero_point(x, tensor_dict[
+        "x_zero_point"]) if "x_zero_point" in node.inputs else tf.cast(
+            x, tf.float32)
+
+    if "w_zero_point" in node.inputs:
+      w_zero_point = tensor_dict["w_zero_point"]
+
+      if w_zero_point.shape.rank == 0:
+        # Simply apply w_zero_point for scalar
+        w = cls._apply_zero_point(w, w_zero_point)
+        result = process_conv(x, w)
+      elif w_zero_point.shape.rank == 1:
+        # Need additional processing for 1d w_zero_point
+        tensor_list = []
+        process_shape = [1] + [w.shape[i] for i in range(1, len(w.shape))]
+        for i in range(w.shape.as_list()[0]):
+          # Apply w_zero_point for each element in 1d tensor
+          out_tensor = cls._apply_zero_point(w[i], w_zero_point[i])
+          new_w = tf.reshape(out_tensor, process_shape)
+          tensor_list.append(process_conv(x, new_w)[0])
```

Comment:
Consider apply w_zero_point to w first then execute Conv with x and the new w. Then you only need to run Conv one time and don't need to do any Reshape afterward.